### ISSUE TITLE
Text Encoder Concept Training seems not using my concept picture I uploaded

### ISSUE THREAD

**Issue by alinasama** [TheLastBen/fast-stable-diffusion#1378]
Hi, I uploaded 600 concept picture mainly showing the fashion model, and set Text_Encoder_Concept_Training_Steps as 4000 step, learning rate keep 1e-6. when I was testing the model after training, the pictures are really random, like you just haven't make any prompt, and gives random objects like cars, tables, even a unknow distorted twisted pattern, nothing even not any element comes from my concept pictures. could you please check with it? Thank you!

**Comment by TheLastBen** [comment]
Concept images are not for direct training, they are supposed to de-overfit the model in case it suffers from too much overfitting.

**Comment by alinasama** [comment]
Thank you so much! However, I just want to know more details, if I uploaded the concept images, but I even didn't select Text_Encoder_Concept_Training, the concept images would still be used during the UNet Training to help de-overfit the model? another question is in that way, should I prepare the captions for concept images?

**Comment by TheLastBen** [comment]
Concept images training don't use captions and if you set Text_Encoder_Concept_Training to 0, the concept images will be ignored
